runtime/stout/src/xcm_config.rs

```diff
@@ -198,30 +197,19 @@ match_types! {
 	};
 }
 
-pub type Barrier = TrailingSetTopicAsId<
-	DenyThenTry<
-		DenyReserveTransferToRelayChain,
-		(
-			TakeWeightCredit,
-			// Expected responses are OK.
-			AllowKnownQueryResponses<PolkadotXcm>,
-			// Allow XCMs with some computed origins to pass through.
-			WithComputedOrigin<
-				(
-					// If the message is one that immediately attemps to pay for execution, then
-					// allow it.
-					AllowTopLevelPaidExecutionFrom<Everything>,
-					// Parent, its pluralities (i.e. governance bodies), and the Fellows plurality
-					// get free execution.
-					AllowUnpaidExecutionFrom<ParentOrParentsExecutivePlurality>,
-					// Subscriptions for version tracking are OK.
-					AllowSubscriptionsFrom<ParentOrSiblings>,
-				),
-				UniversalLocation,
-				ConstU32<8>,
-			>,
-		),
-	>,
+pub type Barrier = DenyThenTry<
```

Comment:
`WithComputedOrigin` is generating issues with `FixedRateOfFungible` `Trader` with `ForeignAssets`. This does not happen to `TrustedAssets` which I have already tested so it doesn't break any previous use case however it does break foreign assets use case and since other teams are using it for testing I would leave it without `WithComputedOrigin` until either this is fixed or we figure out the reason behind it and open a related `issue`